src/fluree/db/query/exec/eval.cljc

```diff
@@ -789,159 +808,92 @@
                          'variable? where/variable?
                          'mch->typed-val where/mch->typed-val}
 
-           json-ld-fns {'expand-iri json-ld/expand-iri}
+           json-ld-fns {'expand-iri json-ld/expand-iri
+                        'json-ld/expand-iri json-ld/expand-iri
+                        'parse-context json-ld/parse-context
+                        'json-ld/parse-context json-ld/parse-context}
```

Comment:
I don't think you need the ns-qualified keys in this map, it's meant to hold symbols that belong in the `json-ld` ns.